plugins/Jobs.cpp

```diff
@@ -220,128 +238,192 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         //     Returns:
         //     - jobID - Unique identifier of this job
         //
-        verifyAttributeSize(request, "name", 1, MAX_SIZE_SMALL);
+        // - CreateJobs (jobs)
+        //
+        //     Creates a list of jobs.
+        //
+        //     Parameters:
+        //     - jobs (json array):
+        //          - name  - An arbitrary string identifier (case insensitive)
+        //          - data  - A JSON object describing work to be done (optional)
+        //          - firstRun - A "YYYY-MM-DD HH:MM:SS" datetime of when
+        //                  this job should next execute (optional)
+        //          - repeat - A description of how to repeat (optional)
+        //          - priority - High priorities go first (optional, default 500)
+        //          - unique - if true, it will check that no other job with this name already exists, if it does it will
```

Comment:
Put the docs for unique and firstrun in one line, or use the same formatting for both